front/app/modules/commercial/custom_maps/utils/tileProviderDefaultOptions.ts

```diff
@@ -0,0 +1,22 @@
+export const MAPTILER_DEFAULT_OPTIONS = {
+  tileSize: 512,
+  zoomOffset: -1,
+  detectRetina: false,
+  minZoom: 1,
+  maxZoom: 19,
+  crossOrigin: true,
+  subdomains: ['a', 'b', 'c'],
+  attribution:
+    '&copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors',
```

Comment:
Do you also want to open this in a new tab?

Reply:
This is copied straight from the examples, but actually it does make sense to open it in a new tab. Will add that :+1: